app/models/site_setting.rb

```diff
@@ -2,10 +2,18 @@
 
 class SiteSetting < Base
   def self.cycle_schedule
-    RedisService.new.get('cycle_schedule')&.to_sym || :real
+    if HostingEnvironment.known_environment?
+      RedisService.new.get('cycle_schedule')&.to_sym || :real
+    else
+      :real
+    end
```

Comment:
@davidgisbey , @duncanjbrown this is an experimental commit - the build is failing due to it trying to connect to redis whilst building. This approach gets around that problem (but has issues of its own).

Any suggestions to improving this and/or alternative approaches?